File: LES_SERIE_S1_IA_FS_ALGO_PYTHON_C/Série_de_TD-TP_3/exercice06.py
def algo1():
    N=int(input("saisir un entier N: "))
    i=0
    while True:
        N=N+1
        print(N)
        i+=1
        if i==10 :
            break
def algo2():
    N=int(input("saisir un entier N: "))
    for i in range(0,10):
        N=N+1
        print(N)
algo1()
algo2()

File: LES_SERIE_S1_IA_FS_ALGO_PYTHON_C/Série_de_TD-TP_3/exercice06.c

#include <stdio.h>
int N,i;
int algo1(){
    printf("saisir un enteir N: ");
    scanf("%d",&N);
    i=0;
    do{
        N++;
        printf("%d\n",N);
        i++;
    }while(i<10);
    return 0;
}
int algo2(){
    printf("saisir un enteir N: ");
    scanf("%d",&N);
    for(int i=0;i<10;i++){
        N++;
        printf("%d\n",N);
    }
    return 0;
}
int main(){
    algo1();
    algo2();
    return 0;
}
